extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -58,6 +61,11 @@ export class AmpAdExit extends AMP.BaseElement {
 
     this.userFilters_ = {};
 
+    /** @private @const {string} */
+    this.creativeId_ = (this.win.frameElement &&
+        this.win.frameElement.getAttribute('data-amp-3p-sentinel')) ||
```

Comment:
This still makes me nervous given we shouldn't even be specifying a sentinel on friendly frames.  Given this is within the AMP creative, which is within the iframe, it should have its own window such that this.getAmpDoc().win is already scoped to the creative so I don't see why this is necessary.

Reply:
Fixed. More thorough testing would not be a bad thing here, though.

Reply:
Ok, tested more thoroughly.
Test 1 = 4 creatives, the first two have one vendor and the second two have a different vendor. Tested with all using the same response keys.
Test 2 = same as test 1, but the two vendors use different response keys.
Test 3 = 1 creative which has amp-analytics and amp-ad-exit for both of the vendors, and like in test 1 the two vendors use the same response keys.
Test 4 = 1 creative which has amp-analytics and amp-ad-exit for both of the vendors, and like in test 2 the two vendors use different response keys.

Reply:
After further testing, it was necessary to add creative ID back. See discussion at https://github.com/ampproject/amphtml/pull/10714#discussion_r136403562